sheets436/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
         minSdkVersion 22
         targetSdkVersion 25
         versionCode 1
-        versionName "0.0.5"
+        versionName "0.0.6-prescription"
```

Comment:
Actually, do you mind changing this to `0.0.7`? I forgot to update the version when I made the `0.0.6` release.